src/middleware/util.js

```diff
@@ -118,9 +119,18 @@ export function csvFileToEntries (file) {
     .replace("'", '')
     .replaceAll('"', '')
     .split(/\r?\n/)
-    .map(row => row.split(','))
+    .map((row) => row.split(','))
```

Comment:
Not sure if this a OSX thing or if prettier/eslint is misconfigured but this callback is an iife and doesn't need to be wrapped.